packages/gdk_pixbuf.rb

```diff
@@ -39,6 +39,6 @@ def self.install
   
   def self.postinstall
     # If the package is installed using a "DESTDIR" method, an important file was not installed and should be copied or generated.
-    system "sudo gdk-pixbuf-query-loaders --update-cache"
+    puts "please run \"sudo gdk-pixbuf-query-loaders --update-cache\" to generate an important file that was not installed."
```

Comment:
Is the sudo command necessary?  What happens if you exclude it?

Reply:
It is necessary. If it is excluded, you can also install it without any errors. But it's highly recommended to run this

Reply:
What exactly does it install and where?  If it only installs in the `/usr/local` directory, the sudo prefix is not needed.

Reply:
I don't know it exactly. The only problem is that we use DESTDIR. If use regular destdir, there is no such a problem.

Reply:
I believe this is what you will want to do:
```
   def self.install
     system "pip install six" # for installation process
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system "mkdir -p #{CREW_DEST_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0"
+    system "touch #{CREW_DEST_LIB_PREFIX}/gdk-pixbuf-2.0/2.10.0/loaders.cache"
     system "pip uninstall --yes six"
   end

   def self.postinstall
-    # If the package is installed using a "DESTDIR" method, an important file was not installed and shoul$
-    puts "please run \"sudo gdk-pixbuf-query-loaders --update-cache\" to generate an important file that $
+    system 'gdk-pixbuf-query-loaders --update-cache'
   end
```

Reply:
Did you get a chance to test this suggestion yet @lovedheart?